index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <title>Todo</title>
+
+    <meta charset="UTF-8">
+    <meta name="description" content="Very simple todo list">
+    <meta name="author" content="Shamim">
+    <meta name="viewport" content="width=device, initial-scale=1.0">
+
+    <link rel="stylesheet" href="./assests/css/index.css">
+    <script src="https://kit.fontawesome.com/ccf39592f4.js" crossorigin="anonymous"></script>
+  </head>
+
+  <body>
+    <main class="main-box">
+      <h1 class="title">Todo</h1>
+      <nav>
+        <ul class="nav-list-box">
+          <li><p>All Tasks 1</p></li>
+          <li><p>Completed 2</p></li>
+          <li><p>Uncompleted 3</p></li>
+        </ul>
+      </nav>
+      <ul class="add-task-box">
```

Comment:
I believe this is not a list. it's just an input.

Reply:
you are right
that was a stupid mistake

Reply:
fixed

Reply:
sorry about that